src/modules/new-request-form/translations/en-us.yml

```
@@ -156,3 +156,13 @@ parts:
       title: "Additional label shown above the credit card field, informing the user that only the last 4 digits are required."
       screenshot: "https://drive.google.com/file/d/13Pz2p5q7FHX9FKT6MUR6Sg5vWy55X0kP/view?usp=drive_link"
       value: "(Last 4 digits)"
+  - translation:
+      key: "new-request-form.lookupfield.loading-options"
```

I would change the key to make it more readable
```suggestion
      key: "new-request-form.lookup-field.loading-options"
```

---

ok, changed